app/controllers/api/go_cardless_controller.rb

```diff
@@ -55,4 +55,8 @@ def payment_options
       session_token: session[:go_cardless_session_id]
     }
   end
+
+  def transaction_options
+    payment_options
```

Comment:
If you see this commit: https://github.com/SumOfUs/Champaign/commit/1c2f187e5999c9c873f6d29128ab3b0fb5d10536, it's added for compatibility with `PaymentController#transaction`. In the case of Braintree payments, we're extending `payment_options` with `store_in_vault`, but since this is Braintree specific, for GoCardless we're just returning `payment_options` and letting the flag default to false (or nil?).